libs/core/src/lib/card/card-header.component.ts

```diff
@@ -0,0 +1,52 @@
+import {
+    Component,
+    OnInit,
+    ChangeDetectionStrategy,
+    ElementRef,
+    ContentChild,
+    Input,
+    HostBinding
+} from '@angular/core';
+
+import { applyCssClass, CssClassBuilder } from '../utils/public_api';
+import { AvatarComponent } from '../avatar/avatar.component';
+
+import { CLASS_NAME } from './constants';
+
+@Component({
+    selector: 'fd-card-header',
+    templateUrl: './card-header.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class CardHeaderComponent implements OnInit, CssClassBuilder {
+    /** @hidden */
+    @Input()
+    @HostBinding('attr.tabindex')
+    tabindex = '0';
+
+    /** @hidden */
+    class: string;
+
+    /** @hidden */
+    @ContentChild(AvatarComponent)
+    _avatar: AvatarComponent;
+
+    /** @hidden */
+    constructor(private _elementRef: ElementRef<HTMLElement>) {}
+
+    /** @hidden */
+    ngOnInit(): void {
+        this.buildComponentCssClass();
+    }
+
+    @applyCssClass
+    /** @hidden */
+    buildComponentCssClass(): string[] {
+        return [CLASS_NAME.cardHeader];
```

Comment:
You can add make `class` property as a Input and put it there [CLASS_NAME.cardHeader, this.class]